src/components/BasicSlider/BasicSlider.tsx

```diff
@@ -36,7 +37,7 @@ const BasicSlider = ({title, value, onChange}: BasicSliderProps) => {
     <Box>
       <SecondaryTitle text={title} sx={{marginBottom: 1}} />
       <Slider
-        sx={styles}
+        sx={{...sx}}
```

Comment:
Why do you need `spread` here? It looks that just `={sx}` should work as well.

Reply:
Yes, it works, thanks.